electives/18792.md

```diff
@@ -0,0 +1,35 @@
+# 18-792 : Advanced Digital Signal Processing
+
+| Category | Difficulty |
+|:-:       | :-:        |
+| HW       | 3          |
+| Exams    | 4          |
+
+This is a course on Advanced Digital Signal Processing and should typically be taken after having done a course on Signals and Systems (18-290) and/or Digital Signal Processing (18-491/18-691) as well as a course on Probability Theory. The course go fairly deep into the topics mentioned below as well as their applications. Homework problems will ask you to implement a lot of these algorithms, usually on speech or music data, and give you a taste of the applications of the topics covered. The signals examined are usually one dimensional signals.
```

Comment:
Do you mind adding a link to 18-290 since it has a guide in this repo?

Reply:
Done in ca1c911c146a17b80ab8a61a67aca297b09bea73!